### DISCUSSION TITLE
Error opening a TLS connection on Azure MS SQL

### DISCUSSION THREAD

**Discussion by mohsinamjad** [prisma/prisma#6792] [Q&A] [answered]
I'm getting this error while running `npx prisma migrate deploy` to deploy migration SQL file on azure database.

Error:
`P1011: Error opening a TLS connection: The TLS settings didn't allow the connection to be established. Please review your connection string. (error: Connection reset by peer (os error 104))`

Connection string is like this
`  DATABASE_URL="sqlserver://somename.database.windows.net:1433;database=dbname;user=name;password=Intech123;encrypt=true;trustServerCertificate=false;"
`

Environment:
Machine: Ubuntu 16.04
Prisma: 2.19.0

Is there something I need to do on my local machine or on cloud settings?   I'm not sure how it can be fixed.

**Comment by pantharshit00** [comment, marked as answer]
Can you try this again with latest version of Prisma? I remember we merged a fix which handles server redirects for mssql driver so this might get automatically fixed when you upgrade.

**Reply by mohsinamjad** [reply]
Yeah, It's fixed in latest version(2.21.2), thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
